### ISSUE TITLE
inappropriate perceptron option warning

### ISSUE THREAD

**Issue by barak** [mlpack/mlpack#1564]
````
$ mlpack_perceptron --input_model_file mlpack_perceptron.bin --test_file test.csv --output_file test-labels1.csv
[WARN ] '--output_file (-o)' ignored because '--test_file (-T)' is specified!
````
Hey, this is exactly like on `mlpack_perceptron(1)`!
````
$ man mlpack_perceptron | egrep 'test_file.*output_file'
       $ perceptron --input_model_file perceptron_model.bin --test_file test_data.csv --output_file predictions.csv
````

**Comment by rcurtin** [comment]
Opened #1566.  Thanks for pointing this out. :+1:

**Comment by rcurtin** [comment]
Fixed since #1566 is merged.  Thanks again for the report. :)
